src/domain/events/eventActionsDropdown/EventActionsDropdown.tsx

```diff
@@ -66,6 +68,17 @@ const EventActionsDropdown: React.FC<EventActionsDropdownProps> = ({
     navigate(eventUrlWithReturnPath);
   };
 
+  const goToEventsPage = () => {
```

Comment:
The abstract features of these `goTo<x>Page` functions seem to repeat between the different components / pages. Maybe there could be just a single shared util function that takes a callback function for initializing `queryString` and navigating to a given path?